src/graph.ts

```diff
@@ -3,10 +3,23 @@
  * Define the factory graph and its components
  * lgfrbcsgo & Nikolaus - October 2020
  */
-import { Craftable, Item } from "./items"
+import { Craftable, Item, Quantity, Liter } from "./items"
 import { findRecipe } from "./recipes"
 
 export type PerMinute = number
+export enum ContainerSize {
+    XS = "XS",
+    S = "S",
+    M = "M",
+    L = "L",
+}
+/* Container volumes sorted from largest to smallest */
+export const ContainerVolume = new Map<ContainerSize, Liter>([
```

Comment:
For consistency, constants should be all upper case.
```suggestion
export const CONTAINER_VOLUME = new Map<ContainerSize, Liter>([
```

Reply:
No longer needed